format-registry/stream/index-respec.html

```diff
@@ -11,7 +11,7 @@
       specStatus: "ED",
 
       // the specification's short name, as in http://www.w3.org/TR/short-name/
-      shortName: "encrypted-media", <!-- The registry does not have its own short name. -->
+      shortName: "eme-stream-registry", <!-- The registry does not have its own short name. -->
```

Comment:
The changes made to the text of format-registry/initdata/index-respec.html should probably be mad here as well.